tap_redshift/__init__.py

```diff
@@ -18,22 +18,20 @@
 # data.world, Inc.(http://data.world/).
 
 import copy
+import datetime
+import ssl
+import sys
 import time
 from itertools import groupby
 
 import pendulum
-import datetime
```

Comment:
The reason these moved is because I used isort to sort them in accordance with pep8.